.github/workflows/benchmark.yml

```diff
@@ -1,68 +1,69 @@
-name: Run Benchmarks
+name: Update benchmarks weights
```

Comment:
imho we could simplify and make more consistent our ci if we would have rule of file = name = label

Update benchmarks weights -> snake -> update-benchmarks-weights.yaml -> kebab -> update-benchmarks-weights-${{ matrix.runtime }}-${{ github.ref }} -> label == update-benchmarks-weights

Reply:
Let's save that for a later PR, but agree